src/Agent.php

```diff
@@ -624,7 +630,7 @@ private function handleAgentResponse(Response $response, $request): array
 
         switch ($request) {
             case self::ACTION_STATUS:
-                $data['answer'] = preg_replace('/status: /', '', $raw_content);
+                $data['answer'] = Sanitizer::sanitize(preg_replace('/status: /', '', $raw_content));
```

Comment:
No need to escape DB special chars.
```suggestion
                $data['answer'] = Sanitizer::sanitize(preg_replace('/status: /', '', $raw_content), false);
```